packages/server/src/utils/buildChatflow.ts

```diff
@@ -54,7 +54,9 @@ export const utilBuildChatflow = async (req: Request, socketIO?: Server, isInter
     try {
         const appServer = getRunningExpressApp()
         const chatflowid = req.params.id
-        const baseURL = `${req.protocol}://${req.get('host')}`
+
+        const httpProtocol = req.get('x-forwarded-proto') || req.protocol
```

Comment:
this doesn't fix it no? `req.get('x-forwarded-proto')` will always be `http` behind proxy, and we want `htttps`

Reply:
actually no, `req.get('x-forwarded-proto')` will be `https` behind proxy and `req.protocol` will be `http`. Have you tested it on AWS @danieldabate ?

Reply:
do we need to check for capital letters as well? `'X-Forwarded-Proto'` ?

Reply:
"Get" method is case insensitive according to my tests, so current implementation works for all cases.

Reply:
awesome thanks!